db_dtypes/pandas_backports.py

```diff
@@ -80,10 +79,16 @@ def _cmp_method(self, other, op):  # pragma: NO COVER
         return NotImplemented
 
 
-# TODO: use public API once pandas 1.5 / 2.x is released.
+# TODO: use public API once NDArrayBackedExtensionArray is added to the
+# pandas.
 # See: https://github.com/pandas-dev/pandas/pull/45544
-@import_default("pandas.core.arrays._mixins", pandas_release < (1, 3))
-class NDArrayBackedExtensionArray(pandas.core.arrays.base.ExtensionArray):
+# Also: Right now none of this is tested in test_pandas_backports.
+# Temporarily marking this as # pragma: NO COVER just to see how this
+# affects unittest coverage
+@import_default("pandas.core.arrays._mixins")
+class NDArrayBackedExtensionArray(
+    pandas.core.arrays.base.ExtensionArray
+):  # pragma: NO COVER
```

Comment:
If we aren't changing the imports in other files, just add the class here like this:

```suggestion
NDArrayBackedExtensionArray = pandas.core.arrays._mixins.NDArrayBackedExtensionArray
```

Reply:
We only referenced `NDArrayBackedExtensionArray` in a single class: `BaseDateTimeArray`.
I updated that class signature directly.